src/utils/bootstrapUtils.js

```diff
@@ -126,6 +127,16 @@ export function getClassSet(props) {
   return classes;
 }
 
+export function omitBsProps(props) {
+  props = { ...props };
+
+  delete props.bsClass;
```

Comment:
this should probably just use lodash omit, deleting properties deoptimizes code paths. Its extremely minor but not reason to not take advantage of lodash here since we have it.